### PR TITLE
docs(render): add informations about render target

### PR DESCRIPTION
**Pull request by haarchri**
### Description of your changes

Fixes #

I have:

- [x] Read and followed Upbound's [contribution process](https://git.io/fj2m9).
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added `backport release-x.y` labels to auto-backport this PR, as appropriate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,18 @@ This repository serves as a foundational step. To enhance your control plane, co
...
+The annotations guide the rendering process, specifying paths to composition, function, environment, and observe files.
+
+### Annotations
+The make render target relies on specific annotations in your YAML files to determine how to process each file.
+The following annotations are supported:
+
+**render.crossplane.io/composition-path**: Specifies the path to the composition file to be used in rendering.
+**render.crossplane.io/function-path**: Specifies the path to the function file to be used in rendering.
+**render.crossplane.io/environment-path** (optional): Specifies the path to the environment file. If not provided, the rendering will proceed without an environment.
+**render.crossplane.io/observe-path** (optional): Specifies the path to the observe file. If not provided, the rendering will proceed without observation settings.
```
Comment (ytsarev) [inline README.md:53]: image
could you check rendering? maybe it's safer to keep newlines between the entries

File: examples/configuration.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: pkg.crossplane.io/v1
 kind: Configuration
 metadata:
-  name: cofiguration-aws-eks
+  name: configuration-aws-eks
 spec:
   package: xpkg.upbound.io/upbound/configuration-aws-eks:v0.12.0
```
Comment (ytsarev) [inline examples/configuration.yaml:6]: can we bump the version here while we have this unique opportunity? :D

File: README.md [hunk 3]
```diff
@@ -36,3 +36,18 @@ This repository serves as a foundational step. To enhance your control plane, co
...
 Upbound will automatically detect the commits you make in your repo and build the configuration package for you. To learn more about how to build APIs for your managed control planes in Upbound, read the guide on Upbound's docs.
+
+# Using the make file
+## render target
+### Overview
+make render target automates the rendering of Crossplane manifests using specified annotations within your YAML files.
+The annotations guide the rendering process, specifying paths to composition, function, environment, and observe files.
+
+### Annotations
+The make render target relies on specific annotations in your YAML files to determine how to process each file.
```
Comment (ytsarev) [inline README.md:47]:
```suggestion
The `make render` target relies on specific annotations in your YAML files to determine how to process each file.
```

File: README.md [hunk 4]
```diff
@@ -36,3 +36,18 @@ This repository serves as a foundational step. To enhance your control plane, co
 
 
 Upbound will automatically detect the commits you make in your repo and build the configuration package for you. To learn more about how to build APIs for your managed control planes in Upbound, read the guide on Upbound's docs.
+
+# Using the make file
+## render target
+### Overview
+make render target automates the rendering of Crossplane manifests using specified annotations within your YAML files.
```
Comment (ytsarev) [inline README.md:43]:
```suggestion
The `make render` target automates the rendering of Crossplane manifests using specified annotations within your YAML files.
```

File: README.md (hunk 1 above)
Comment (ytsarev) [reply README.md:53]: Can we also provide the embedded yaml example of usage for clarity?

**Review by ytsarev — COMMENTED** [review]
left some suggestions, otherwise looks good

File: examples/configuration.yaml (hunk 2 above)
Comment (haarchri) [reply examples/configuration.yaml:6]: yes its the new version

**Review by ytsarev — APPROVED** [review]
LGTM

**Pull request merged**